AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.ts

```diff
@@ -496,9 +531,47 @@ export class AddParticipantComponent extends BookingBaseComponent implements OnI
         return this.showDetails && this.searchEmail ? this.searchEmail.validateEmail() : true;
     }
 
+    validateJudgeAndJohMembers(): boolean {
+        if (this.hearing?.participants.length) {
+            const judge = this.hearing.participants.find(x => x.is_judge);
+            return this.searchEmail.email !== judge?.username;
+        }
+        return true;
+    }
+
+    validateJudiciaryEmailAndRole() {
+        if (this.searchEmail && this.searchEmail.email.length) {
+            this.searchService.searchJudiciaryEntries(this.searchEmail.email).subscribe(x => {
```

Comment:
I think the searchEmail component should do this searching.

Reply:
These changes are from VIH-7284 which is already merged into master. This branch was based on that branch before it was merged into master. I didn't expect commits from that branch to show up in this pull request.

This is called when the role is changed to ensure the user that was previously searched it is still valid. It probably should be refactored to go through the searchEmail component though.